ansible-eda-base-image/Containerfile

```diff
@@ -0,0 +1,25 @@
+FROM registry.access.redhat.com/ubi9-minimal
```

Comment:
I thought that splitting the build into a base image with most dependencies, then a second build that contains the specific rules and playbooks could be a nice way to speed up builds.